### PR TITLE
Add pallete based on xgfs_normal6 from Anton Tsitsulin

### PR DESCRIPTION
**Pull request by martinkleppe**
This commit will add the normal pallette based on xgfs_normal6 from Anton Tsitsulin as described in issue #2010.

### CODE REVIEW

**Review by swharden — APPROVED** [review]

**Comment by swharden** [comment]
Thanks @martinkleppe, this looks great! I'll merge this after @bclehmann approves too 👍

**Review by bclehmann — APPROVED** [review]
We should probably add this to ScottPlot 5 as well, but that can be in a later PR.

**Pull request merged**
